core/block_drag_surface.js

```diff
@@ -179,8 +178,9 @@ Blockly.BlockDragSurfaceSvg.prototype.createDropShadowDom_ = function(defs) {
  * surface.
  */
 Blockly.BlockDragSurfaceSvg.prototype.setBlocksAndShow = function(blocks) {
-  goog.asserts.assert(
-      this.dragGroup_.childNodes.length == 0, 'Already dragging a block.');
+  if (this.dragGroup_.childNodes.length) {
```

Comment:
This would be clearer to read/understand if it were: `if (this.dragGroup_.childNodes.length > 0)`

Reply:
Just following the Google JS Style Guide which refers to this as "shorter and nicer" (under "Tricks and Tips).  It is the same way that the Closure Library is written:
https://github.com/google/closure-library/blob/master/closure/goog/color/color.js#L116